src/components/NavigationDrawer.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { StyleSheet, View, Image } from 'react-native'
+import { Button, Text } from 'native-base'
+import BottomSheet from 'reanimated-bottom-sheet'
+import Icon from './ShabadOsIcon'
+
+import logo from '../../assets/logo.png'
+
+type NavigationBarProps = {
+    navigation: StackNavigationProp<>;
+  }
+
+const styles = StyleSheet.create( {
+  navigationPanel: {
+    height: '100%',
+    backgroundColor: '#f9f9fa',
+    opacity: 0.92,
+  },
+  navigationPanelHeader: {
+    backgroundColor: '#f9f9fa',
+    paddingTop: 7,
+    borderTopLeftRadius: 20,
+    borderTopRightRadius: 20,
+    alignItems: 'center',
+    opacity: 0.92,
+  },
+  navigationPanelHandle: {
+    width: 80,
+    height: 2,
+    borderRadius: 5,
+    backgroundColor: '#00000040',
+  },
+  navigationPanelIconContainer: {
+    flexDirection: 'row',
+    paddingTop: '5%',
+    paddingBottom: '10%',
+  },
+  navigationIconSpacing: {
+    paddingRight: 30,
+    paddingLeft: 30,
+  },
+  navigationPanelButtonContainer: {
+    paddingTop: '1%',
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+  },
+} )
+
+const Buttons = [
+  [ 'history', 'WIP' ],
+  [ 'bookmark', 'WIP' ],
+  [ 'search', 'Search' ],
+  [ 'tabs', 'WIP' ],
+  [ 'ellipsis', 'WIP' ],
+]
+
+const NavigationBar: React.FC<NavigationBarProps> = ( { navigation } ) => {
+  const renderNavigationHeader = () => (
+    <View style={styles.navigationPanelHeader}>
+      <View style={styles.navigationPanelHandle} />
+      <View style={styles.navigationPanelIconContainer}>
+        {Buttons.map( ( [ iconName, destination ] ) => (
+          <View key={iconName} style={styles.navigationIconSpacing}>
+            <Icon
+              name={iconName}
+              size={20}
+              color="rgb(47,124,246)"
+              onPress={() => navigation.navigate( destination )}
+            />
+          </View>
+        ) )}
+      </View>
+    </View>
+  )
+
+  const renderNavigationContent = () => (
+    <View style={styles.navigationPanel}>
```

Comment:
This should go inside a `<FooterTab>` component 

https://nativebase.io/docs/v0.5.9/customize#themeFooterTabs

Reply:
That is used for rendering inside the `BottomSheet` that is from [`reanimated-bottom-sheet`](https://github.com/osdnk/react-native-reanimated-bottom-sheet). I don't think I should be using `FooterTab` because it makes it as it's own thing inside. 
![image](https://user-images.githubusercontent.com/44710980/79775921-8ef3d200-82fa-11ea-8293-d8f9ad731ae2.png)